webui/react/src/e2e/models/pages/Admin/UserManagement.ts

```diff
@@ -177,8 +180,11 @@ class UserRow extends Row {
     selector: '[data-testid="modified"]',
   });
   readonly actions = new UserActionDropdown({
-    parent: this,
-    selector: '[data-testid="actions"]',
+    childNode: new BaseComponent({
+      parent: this,
+      selector: '[data-testid="actions"]',
```

Comment:
can this selector be a static constant since it's used twice in the same file?

Reply:
no. they're different buttons that I gave the same ID. they don't conflict because of our framework's robustness, but they aren't the same.